resources/validations/src/ssp.sch

```diff
@@ -438,43 +408,1422 @@
             <sch:let name="extraneous-parties"
                      value="$responsible-parties[not(o:party-uuid = $parties/@uuid)]" />
-            <sch:assert id="incorrect-role-association"
+            <sch:assert diagnostics="incorrect-role-association-diagnostic"
                         doc:organizational-id="section-c.6"
-                        test="not(exists($extraneous-roles))">[Section C Check 2] This SSP has defined a responsible party with 
-            <sch:value-of select="count($extraneous-roles)" />
-            <sch:value-of select="if (count($extraneous-roles)=1) then ' role' else ' roles'" />not defined in the role: 
-            <sch:value-of select="$extraneous-roles/@role-id" /></sch:assert>
-            <sch:assert id="incorrect-party-association"
+                        id="incorrect-role-association"
+                        role="error"
+                        test="not(exists($extraneous-roles))">[Section C Check 2] This SSP has defined a responsible party with no extraneous
+                        roles.</sch:assert>
+            <sch:assert diagnostics="incorrect-party-association-diagnostic"
                         doc:organizational-id="section-c.6"
-                        test="not(exists($extraneous-parties))">[Section C Check 2] This SSP has defined a responsible party with 
-            <sch:value-of select="count($extraneous-parties)" />
-            <sch:value-of select="if (count($extraneous-parties)=1) then ' party' else ' parties'" />is not a defined party: 
-            <sch:value-of select="$extraneous-parties/o:party-uuid" /></sch:assert>
+                        id="incorrect-party-association"
+                        role="error"
+                        test="not(exists($extraneous-parties))">[Section C Check 2] This SSP has defined a responsible party with no extraneous
+                        parties.</sch:assert>
         </sch:rule>
         <sch:rule context="/o:system-security-plan/o:back-matter/o:resource">
-            <sch:assert id="resource-uuid-required"
+            <sch:assert diagnostics="resource-uuid-required-diagnostic"
                         doc:organizational-id="section-b.?????"
-                        test="./@uuid">[Section B Check ????] This SSP includes back-matter resource missing a UUID</sch:assert>
-        </sch:rule>
-        <sch:rule context="/o:system-security-plan/o:back-matter/o:resource/o:rlink">
-            <sch:assert id="resource-rlink-required"
-                        doc:organizational-id="section-b.?????"
-                        test="doc-available(./@href)">[Section B Check ????] This SSP references back-matter resource: 
-            <sch:value-of select="./@href" /></sch:assert>
+                        id="resource-uuid-required"
+                        role="error"
+                        test="./@uuid">[Section B Check ????] This SSP has back-matter resources each with a UUID.</sch:assert>
```

Comment:
Can we appropriately modify these section identifiers to label `Section B Check ????` with the right `????` please? (This is the first of a half dozen or so in the code base.)

Reply:
@GaryGapinski that seems like an important idea to capture. Can we discuss and come to back to this in a later story then? Since this is not specific to attachment `back-matter/resource` elements, can you just remove `[Section B Check ????]` altogether and we will revisit?

Reply:
I'll remove them in advance of a decision to have a modified checklist for cross-reference.

Reply:
Most recently (i.e., this morning) the `[Section B Check ????]` prefixes have been removed.